content/extensions.md

```diff
@@ -69,6 +69,6 @@ To create an own extension,
 
 | Extension       | Extension Identifier      | Description                                                                                         | Document   |
 |-----------------|---------------------------|-----------------------------------------------------------------------------------------------------|------------|
-| Legacy Stats    | org.homie.legacy-stats    | This extension adds the stats functionality of Homie `3.0.1` to Homie `4.0`                         | [GitHub]() |
-| Legacy Firmware | org.homie.legacy-firmware | This extension adds the firmware, mac and localip device attributes of Homie `3.0.1` to Homie `4.0` | [GitHub]() |
-| Meta            | eu.epnw.meta              | This extension defines how to add metadata and tags to devices, nodes and properties                | [GitHub]() |
+| Legacy Stats    | org.homie.legacy-stats    | This extension adds the stats functionality of Homie `3.0.1` to Homie `4.0`                         | [GitHub](https://github.com/homieiot/convention/blob/develop/extensions/documents/homie_legacy_stats_extension.md) |
```

Comment:
This link doesn't work

Reply:
Yes, it will work after the homieiot/convention#172 is merged.